### ISSUE TITLE
llvmpipe crashes on minimal integer division by -1

### ISSUE THREAD

**Issue by AaronGhost** [AaronGhost/glslsmith-bugs-tracker#1]
# Description

A shader containing only the division of the minimal int by -1 crashes with llvmpipe outputing the following error.
```glsl
#version 450
layout(local_size_x = 1, local_size_y = 1, local_size_z = 1) in;
layout(std140, binding = 0) buffer buffer_0 {
  int x;
};

void main()
{
  x = -2147483648/-1;
}
```

# llvmpipe output
```code
Floating point exception (core dumped)
```

# Thinking process
From what I understand from the GLSL ES 3.1 specification (4.1.3 Integers), it should return an undefined value but not crash.

For the related code in the shadertrap  test format, see: [llvmpipe/crashes/shader01.shadertrap](../blob/master/llvmpipe/crashes/shader01.shadertrap)
For the related code in the shader_runner test format, see: [llvmpipe/crashes/shader01.shader_test](../blob/master/llvmpipe/crashes/shader01.shader_test)

**Comment by AaronGhost** [comment]
Fixed in mesa-21.1.2 without reporting

**Comment by AaronGhost** [comment]
Reported to the mesa project: https://gitlab.freedesktop.org/mesa/mesa/-/issues/4879
